packages/vite/src/node/ssr/__tests__/ssrTransform.spec.ts

```diff
@@ -133,8 +133,7 @@ test('export then import minified', async () => {
     ),
   ).toMatchInlineSnapshot(`
     "const __vite_ssr_import_0__ = await __vite_ssr_import__(\\"vue\\", {\\"namedImportSpecifiers\\":[\\"createApp\\"]});
-    const __vite_ssr_import_1__ = await __vite_ssr_import__(\\"vue\\", {\\"isExportAll\\":true});
-    __vite_ssr_exportAll__(__vite_ssr_import_1__);
+    __vite_ssr_exportAll__(__vite_ssr_import_0__);
```

Comment:
This change was what I'm most concern of with my PR before. Besides `isExportAll` missing here (actually it seems like we don't need the property at all, I'll fix that), this PR would break something like:


```js
import { someApi } from 'lib'
import { anotherApi } from 'lib'
```

Where the `namedImportSpecifiers` won't be merged. Even if we solved this by merging implicitly, I'm still a bit concern if we have to add more metadata in the future, plus the execution order is already incorrect and fixing it might be harder after this PR 🤔

Reply:
Oh, You have a point. This PR looks worthless currently.

Reply:
Well it's great that we explore this though, so not worthless! But it might only be an optimization for a later time.